src/pyhf/pdf.py

```diff
@@ -319,6 +319,36 @@ def par_slice(self, name):
         """
         return self.par_map[name]['slice']
 
+    def par_names(self, fstring='{name}[{index}]'):
+        """
+        The names of the parameters in the model including binned-parameter indexing.
+
+        Args:
+            fstring (:obj:`str`): Format string for the parameter names using `name` and `index` variables. Default: '{name}[{index}]'.
```

Comment:
```suggestion
            fstring (:obj:`str`): Format string for the parameter names using ``name`` and ``index`` variables. Default: ``'{name}[{index}]'``.
```

For the render https://pyhf--1536.org.readthedocs.build/en/1536/_generated/pyhf.pdf._ModelConfig.html#pyhf.pdf._ModelConfig.par_names I think these should be using the double ticks to look right